peerannot/runners/datasets.py

```diff
@@ -25,20 +25,123 @@
     "path",
     type=click.Path(),
 )
-def install(path):
+@click.option(
+    "--answers_format",
+    default=0,
+    type=click.INT,
+    help="annotation file format",
+    # 0 == Rodriguez matrix format, 1 == JSON answers/worker format, 2 == JSON worker/answers format
+)
+@click.option(
+    "--answers",
+    default="",
+    type=click.Path(exists=False),
+    help="annotation file",
+)
+@click.option(
+    "--metadata",
+    default="",
+    type=click.Path(exists=False),
+    help="metadata information file",
+)
+@click.option(
```

Comment:
in args/options, we use `labels-path` and not `labels_path` in the CLI. Click standardizes the `-` into `_` on its own later. The dash notation is just more standard (cf [here](https://stackoverflow.com/questions/1253679/should-command-line-options-in-posix-style-operating-systems-be-underscore-style)), same for others